CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.0.7
+  - Changed Fixnum to Integer [#10](https://github.com/logstash-plugins/logstash-filter-sleep/pull/10)
```

Comment:
It is useful to add a reason here - something like "changed use of deprecated Fixnum to Integer to remove warnings in the log"